outline/02_simple_values.md

```diff
@@ -8,14 +8,19 @@ permalink: /outline/simple_values.html
 
 {% comment %}
 
-http://clojurebridge.github.io/curriculum/outline/simple_values.html
+http://clojurebridge-berlin.github.io/curriculum/outline/simple_values.html
 
 {% endcomment %}
 
 <section>
 Simple Values
 ----------------------------------------
 {: .slide-title .chapter}
+</section>
+
+
+<section>
+### Values you can use
```

Comment:
Now that this is a slide on its own that doesn't seem like a title slide, might a coach get confused and start to actually explain each bullet point, rather than realising they're just introducing the headings?

Reply:
OK, per discussion at orga-meeting last night, this PR is good to go pending a double-check on this slide, which I just did, so fasten your seatbelts